src/zodbbrowser/check.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python
+import logging
+import optparse
+import os
+import sys
+import tempfile
+
+from zodbbrowser.standalone import open_database
+from zodbbrowser.references import ReferencesDatabase
+
+
+def iter_database(db):
+    """Iter over records located inside the database.
+    """
```

Comment:
[PEP-257](https://www.python.org/dev/peps/pep-0257/#one-line-docstrings) says single-line docstrings should have the closing `"""` on the same line.